CMakeLists.txt

```diff
@@ -233,14 +233,6 @@ else()
   set(_gRPC_PROTOBUF_LIBRARY_NAME "libprotobuf")
 endif()
 
-if(gRPC_BACKWARDS_COMPATIBILITY_MODE)
-  add_definitions(-DGPR_BACKWARDS_COMPATIBILITY_MODE)
-  if(_gRPC_PLATFORM_MAC)
-    # some C++11 constructs not supported before OS X 10.10
-    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.10)
```

Comment:
what is going to be the CMAKE_OSX_DEPLOYMENT_TARGET value from now on?

Reply:
It seems that you added it by https://github.com/grpc/grpc/pull/16777. I guess this should exist no matter `gRPC_BACKWARDS_COMPATIBILITY_MODE` is true or not.